openapi-ls-extension/src/main/java/io/ballerina/openapi/extension/OpenAPIConverterResponse.java

```diff
@@ -32,7 +31,8 @@
     @Deprecated
     private String yamlContent;
 
-    private List<OASResult> content;
+    // Json Array of OASResults
+    private JsonArray content;
```

Comment:
Would it be okay to change the response type without deprecating the existing API or creating a new API? This will be a breaking change for the existing users.

Reply:
@NipunaMarcus confirmed that there is no current usage in this API and we can change that.

Reply:
currently this is been used only with beta3 ... but this will be used mostly only for this going forward hence I think we don't have to maintain this... but yeah we can deprecate the old and give add a new API but I don't see a point doing that though ...

Reply:
What's the reason for converting the return type to a `JsonArray`? This is too generic in the API point of view, in the sense we expose an API which we can return a JsonArray of any type of elements. Also, isn't the IIRC, the JSON-RPC layer converts the list to a JsonArray under the hood.

Reply:
there was a serialization issue with the type used in List nadeeshaan... it cannot serialize this List hence we decided to to go with a json Array

Reply:
The JSON-RPC layer failed to convert this list and threw an exception before. Because of that we changed it to a json.

Reply:
I think we need to figure out what's the actual serialisation issue first. And then we can expose a proper data model, may be a new data model as the response.

Reply:
As we discussed offline, this can be considered as a special use case that we need to pass the data as an Array since the third party library doesn't support the serialization.